### PR TITLE
Feature/jb 346 jb 347 cleanup venv install

### PR DESCRIPTION
**Pull request by sei-amellinger**
Key changes:
- Remove Pipfile
- Add "getting started" which shows how to start a docker container OR make a venv. (Apple Silicon Not Supported)
- Moves the docker readme into the new docs directory
- Reworks enter_juneberry_container to use switches for workspace and container

### CODE REVIEW

File: docker/enter_juneberry_container [hunk 1]
```diff
@@ -32,24 +32,62 @@
...
+GPUS="--gpus all"
+# GPUS=""
+# GPUS="--gpus \"device=2,3\""
+# GPUS="--gpus \"device=${CUDA_VISIBLE_DEVICES}\""
+
+function show_help() {
+  echo "Usage: [-w <workspace>] [-c <container>] [-g <gpus>] <project>"
+  echo "Optional arguments:"
+  echo "    -w <workspace> - Optional: The path to a custom workspace."
+  echo "    -c <container> - Optional: Name of the container. The defuult is ${CONTAINER}."
```
Comment (sei-wshaw) [inline docker/enter_juneberry_container:51]: "defuult" -> "default"

File: docs/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+Change _into_ the project directory and run the enter_juneberry_container script. By default, the script 
+will attempt to use a locally built `juneberry/cudadev:dev` container. If you downloaded
+a different container, you can specify which container to use via the second argument to the script.
+
+For example, you can use the following command to start an instance of the downloaded CPU-only container:
+
+```shell script
+docker/enter_juneberry_container -c amellinger/juneberry:cpudev <project_directory>
+```
+
```
Comment (sei-wshaw) [inline docs/getting_started.md:81]: If we're saying a user should cd to the project directory to start the container, should we assume <project_directory> is the current directory if they don't provide one?

File: docker/enter_juneberry_container (hunk 1 above)
Comment (sei-amellinger) [reply docker/enter_juneberry_container:51]: Cool!  Feel free to change it. :)

File: docs/getting_started.md (hunk 2 above)
Comment (sei-amellinger) [reply docs/getting_started.md:81]: Yeah, that should probably be `pwd`.

File: docker/enter_juneberry_container (hunk 1 above)
Comment (sei-amellinger) [reply docker/enter_juneberry_container]: Fixed.

File: docs/getting_started.md (hunk 2 above)
Comment (sei-amellinger) [reply docs/getting_started.md:82]: Fixed.

File: docs/getting_started.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+Assuming the same project-root as before, the difference that a manual installation will introduce will be 
+another directory inside the project-root. Perform this installation **INSIDE** the project-root.
+
+## Set up a virtual environment
+
+Any virtual environment,such as venv or pyenv, can be used. The first step is to construct and enter 
+the virtual environment. The following example demonstrates pyenv/venv. 
+
+```shell script
+pyenv virtualenv 3.8.5 jb
```
Comment (sei-nwinski) [inline docs/getting_started.md:159]: Do you think I should add a `pyenv install 3.8.5` above this line? I didn't have it.

Comment (sei-amellinger) [reply docs/getting_started.md:159]: Sure! I'm not sure where the boundary is with this. Should we also have a link for how to install pyenv?

Comment (sei-nwinski) [reply docs/getting_started.md:159]: That was something I also had to do and I have the link so I'll add something.

File: docs/getting_started.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+git clone https://github.com/cmu-sei/juneberry.git
+```
+
+## Install from Requirements
+
+By default, pip does NOT honor the ordering. So the requirements are provided one at a time.
+
+```shell script
+cd juneberry
+cat requirements.txt | xargs pip install
```
Comment (sei-nwinski) [inline docs/getting_started.md:180]: Making a note here that we may need a CPU-only version of the requirements.txt due to onnxruntime-gpu.

**Review by sei-nwinski — APPROVED** [review]

**Pull request merged**